src/app/destinyTrackerApi/d2-reviewsFetcher.ts

```diff
@@ -15,14 +15,13 @@ import { toUtcTime } from './util';
 import { getReviews, getItemReviewsKey } from '../item-review/reducer';
 import { reviewsLoaded } from '../item-review/actions';
 import { ThunkResult } from '../store/reducers';
-import { BungieMembershipType } from 'bungie-api-ts/user';
 
 /**
  * Redux action that populates community (which may include the current user's) reviews for a given item.
  */
 export function getItemReviewsD2(
   item: D2Item,
-  platformSelection: BungieMembershipType,
+  platformSelection: number,
```

Comment:
Can you make a new enum for this instead of using number?

Reply:
That (and the renaming) are good ideas. I'll get them in later today.